README.md

```
@@ -1 +1,56 @@
+<img width="1000" height="1530" alt="image" src="https://github.com/user-attachments/assets/7c52feab-d94e-432a-bc3f-453d6f902e14" />
+
+### Application
+<img width="1000" height="1692" alt="image" src="https://github.com/user-attachments/assets/52f80fd5-c77a-43b8-a988-159e0c8866c6" />
+
+
+## 👥 멤버
+
+### 프론트엔드
+
+
+| <img src="https://github.com/user-attachments/assets/c0694fc2-3078-4417-ba7b-2f7a66af1cc8" width="130" height="130"> | <img src ="https://github.com/user-attachments/assets/f95731c4-2cd3-41f4-9d9b-b695bc48b372" width="130" height="130"> | <img src ="https://github.com/user-attachments/assets/b2325a15-4771-48c2-b1a8-52217f4ee92b" width="130" height="130"> |
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)